### ISSUE TITLE
sql_patent testing fails on module

### ISSUE THREAD

**Issue by doolin** [funginstitute/patentprocessor#5]
This is probably very easy to fix, haven't looked into it yet.

``` sh
Processing test_sql_patent.py file..
Traceback (most recent call last):
  File "test_sql_patent.py", line 13, in <module>
    from sql_patent import SQLPatent
ImportError: No module named sql_patent
```

**Comment by gtfierro** [comment]
I believe `sql_patent` was one of the files that was brought into the test directory in an attempt to refactor the code; as a result, `test_sql_patent.py` didn't actually test the production code. I'm planning on refactoring the file to change this, and there's already a [PT story](https://www.pivotaltracker.com/projects/557581#!/stories/42007907).

**Comment by doolin** [comment]
Perfect.

**Comment by gtfierro** [comment]
This should be fixed as of 8e7636c
